resources/conanfile.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy, update_conandata
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.58.0 <2.0.0"
+
+
+class CuraResource(ConanFile):
+    name = "cura_resources"
+    license = ""
+    author = "UltiMaker"
+    url = "https://github.com/Ultimaker/cura-private-data"
+    description = "Cura Resources"
+    topics = ("conan", "cura")
+    exports = "LICENSE*"
+    settings = "os", "compiler", "build_type", "arch"
+    no_copy_source = True
+
+    def set_version(self):
+        if not self.version:
+            self.version = self.conan_data["version"]
+
+    def export(self):
+        copy(self, pattern="conandata.yml", src=os.path.join(self.recipe_folder, ".."), dst=self.export_folder,
+             keep_path=False)
+        update_conandata(self, {"version": self.version})
+
+    def export_sources(self):
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "definitions"),
+             dst=os.path.join(self.export_sources_folder, "definitions"))
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "extruders"),
+             dst=os.path.join(self.export_sources_folder, "extruders"))
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "images"),
+             dst=os.path.join(self.export_sources_folder, "images"))
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "intent"),
+             dst=os.path.join(self.export_sources_folder, "intent"))
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "meshes"),
+             dst=os.path.join(self.export_sources_folder, "meshes"))
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "quality"),
+             dst=os.path.join(self.export_sources_folder, "quality"))
+        copy(self, pattern="*", src=os.path.join(self.recipe_folder, "variants"),
+             dst=os.path.join(self.export_sources_folder, "variants"))
```

Comment:
I would suggest to define this list of subfolders globally and use loops to process them. As far as I can see, they are used in 3 different places so this is very error-prone.